README.md

```diff
@@ -139,6 +139,36 @@ only really makes sense if you're calling Grover directly (and not via middlewar
 Grover.new('<some URI with basic authentication', username: 'the username', password: 'super secret').to_pdf
 ```
 
+#### Adding cookies
+If you need to set cookies to your new page instance, just pass them along like this (please note that the only required properties are `name` and `value`. Check [.setCookies](https://github.com/puppeteer/puppeteer/blob/master/docs/api.md#pagesetcookiecookies) docs):
```

Comment:
Please limit line length to 120 chars max. Ideally keep it under 100.

Also, suggest using a more formal tone when writing documentation.

```suggestion
To set request cookies when requesting a URL, pass an array of hashes as such

_N.B._ Only the `name` and `value` properties are required. See
[page.setCookies](https://github.com/puppeteer/puppeteer/blob/master/docs/api.md#pagesetcookiecookies)
documentation for more details.
```